client/src/components/WorkflowInvocationState/WorkflowInvocationOverview.vue

```diff
@@ -56,6 +65,7 @@ const uniqueMessages = computed(() => {
         </BAlert>
         <div v-else-if="workflow && !isSubworkflow">
             <InvocationGraph
+                ref="invocationGraph"
```

Comment:
Can you pass activeNodeId as a prop, or probably cleaner, set the active node id via workflowStateStore ?